api/api-facade/api-io/src/main/java/org/eclipse/dirigible/api/v3/io/ZipFacade.java

```diff
@@ -19,17 +23,31 @@
 import java.util.zip.ZipInputStream;
 import java.util.zip.ZipOutputStream;
 
-import org.apache.commons.io.IOUtils;
-import org.eclipse.dirigible.commons.api.helpers.BytesHelper;
-import org.eclipse.dirigible.commons.api.helpers.GsonHelper;
-
 /**
  * Facade for working with ZIP files
  */
 public class ZipFacade {
-	
+
+	/**
+	 * Unzip zip archive from proivided zipPath to targetPath folder
+	 * @param zipPath
+	 * @param targetPath
+	 */
+	public static void importZip(String zipPath, String targetPath) {
```

Comment:
Should we allow the `import/export` zip functionality only for files/folders under the `registry/public` path or for any path (as of now)?

Reply:
in discussion with @vmutafov, we decided for now to allow any path. Of course, if there is no reason to limit yourself to "registry/public"